src/app/api-interactions.ts

```diff
@@ -0,0 +1,76 @@
+import _ from 'lodash';
+import { SeriesVolume, Comic, MarvelAPISeriesResponse } from './models';
+import { HttpParams, HttpClient } from '@angular/common/http';
+
+const MARVEL_API_BASE_URL = 'https://gateway.marvel.com/v1/public/';
+const MARVEL_API_KEY_PUBLIC = '46a863fa31f601aacb87dae9cb8f7c45';
+
+const GOODREADS_API_BASE_URL = 'https://www.goodreads.com/search/index.xml';
+const GOODREADS_API_KEY_PUBLIC = 'ruoM3jpamOVNpjOnfiAuYA';
+
+const CORS_ANYWHERE_URL = 'https://cors-anywhere.herokuapp.com/';
+
+export class ApiInteractions {
+  constructor(
+    public http: HttpClient,
+  ) { }
+
+  /**
+   * @param seriesVolume the series volume object
+   * @returns the series data from the Marvel API
+   */
+  public getAPISeriesVolume = (seriesVolume: SeriesVolume) => {
+    const params = new HttpParams()
+      .set('title', seriesVolume.title)
+      .set('startYear', seriesVolume.startYear)
+      .set('apikey', MARVEL_API_KEY_PUBLIC);
+
+    return this.http.get(MARVEL_API_BASE_URL + 'series', {params});
+  }
+
+  /**
```

Comment:
You recently made edits to multiline comments to use a single asterisk style `/*` but these are doubled.  Is there a reason for that?

Reply:
The double asterisk indicates it's a JSDoc and therefore uses things like `@param` so this is correct

Reply:
Ah, I see!  Will merge